packages/fxa-settings/src/components/images/index.tsx

```diff
@@ -6,9 +6,9 @@ import { ReactComponent as TwoFactorAuth } from './graphic_two_factor_auth.svg';
 import { ReactComponent as PushFactorAuth } from './graphic_push_factor_auth.svg';
 import { ReactComponent as Mail } from './graphic_mail.svg';
 import { ReactComponent as SecurityShield } from './graphic_security_shield.svg';
-import { ReactComponent as Key } from './graphic_key.svg';
+import { ReactComponent as Key } from './graphic_recovery_key.min.svg';
 import { ReactComponent as Lock } from './graphic_lock.svg';
-import { ReactComponent as Lightbulb } from './graphic_lightbulb.svg';
+import { ReactComponent as Lightbulb } from './graphic_recovery_key_hint.min.svg';
```

Comment:
Kept original SVG file (easier to edit) and an SVGO optimized version (`*.min.svg`)

Reply:
I'm curious how other folks would feel about this naming convention, I don't think we're doing anything automatically off of our SVGs file names so I think this could work for us to make it obvious it was sent through SVGO. 👍